experimental/perf/scripts/testfiles/bench_run_log.py

```diff
@@ -0,0 +1,107 @@
+import argparse
+from datetime import datetime
+
+from helper import generate_random_dict
+
+import wandb
+
+
+def measure_time(func):
+    """Decorator to measure and return execution time of a function."""
+
+    def wrapper(*args, **kwargs):
+        start_time = datetime.now()
+        result = func(*args, **kwargs)
+        end_time = datetime.now()
+        elapsed_time = end_time - start_time
+        print(f"{func.__name__}() time: {elapsed_time.total_seconds()}")
+        return result, elapsed_time.total_seconds()
+
+    return wrapper
+
+
+@measure_time
+def init_wandb(run_id, args):
+    """Initialize a new W&B run."""
+    wandb.init(
+        project="perf-test",
+        name=f"perf_{run_id}_{args.steps}steps_{args.metric_count}mc",
```

Comment:
what do you think of something more explicit like this:
```suggestion
        name=f"perf_run={run_id}_steps={args.steps}_metric_count={args.metric_count}",
```